### PR TITLE
Fix Sage theme SVG refresh

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/issues/2375
`h-2,c` is not OK: It should be either `h-2 c` or `h-2c`

### CODE REVIEW

File: p/themes/Swage/icons/refresh.svg [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg enable-background="new 0 0 16 16" version="1.1" viewBox="0 0 16 16" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
 <style type="text/css">.st0{fill:#FFFFFF;}</style>
-<path class="st0" d="M13.5,2.4C12.1,1,10.1,0.1,8,0.1c-4.3,0-7.7,3.5-7.7,7.7s3.5,7.7,7.7,7.7c3.6,0,6.6-2.5,7.5-5.8h-2,c-0.8,2.3-2.9,3.9-5.5,3.9c-3.2,0-5.8-2.6-5.8-5.8S4.8,2.1,8,2.1c1.6,0,3,0.7,4.1,1.7L9,6.9h6.8V0.1L13.5,2.4z"/>
+<path class="st0" d="M13.5,2.4 C12.1,1,10.1,0.1,8,0.1 c-4.3,0-7.7,3.5-7.7,7.7 s3.5,7.7,7.7,7.7 c3.6,0,6.6-2.5,7.5-5.8 h-2 c-0.8,2.3-2.9,3.9-5.5,3.9 c-3.2,0-5.8-2.6-5.8-5.8 S4.8,2.1,8,2.1 c1.6,0,3,0.7,4.1,1.7 L9,6.9 h6.8 V0.1 L13.5,2.4 z"/>
```
Comment (Frenzie) [inline p/themes/Swage/icons/refresh.svg]: Interesting, my comma comment came from the DOM as displayed by Firefox but actually there was no separator.

Superfluous whitespace can be eliminated, so I guess that means Fx concludes this whitespace is not in fact superfluous.

Comment (Alkarex) [reply p/themes/Swage/icons/refresh.svg]: Ah, I found the more precise error. Updated patch coming. There was a clearly wrong comma.

Comment (Alkarex) [reply p/themes/Swage/icons/refresh.svg]: https://github.com/FreshRSS/FreshRSS/pull/2376/commits/a44f65892b27003d5b172e723ea3f67f6c8700fd

Comment (Alkarex) [reply p/themes/Swage/icons/refresh.svg]: `h-2,c` is not OK: It should be either `h-2 c` or `h-2c`

**Pull request merged**